### DISCUSSION TITLE
New sector stuck at committing

### DISCUSSION THREAD

**Discussion by kuchaguangjie** [filecoin-project/lotus#6493] [Storage Provider]
**Describe the bug**

I compiled & installed lotus with tag `v1.9.0`, the **nerpa** testnet.
After creating a 512Mb sector for a day, my `lotus-miner sector list` always shows:

<pre>
ID  State       OnChain  Active  Expiration  Deals  
0   Committing  NO       NO      n/a         CC    
</pre>

And I saw following warning in lotus's log:

> 2021-06-14T12:15:36.678+0800	WARN	miner	miner/miner.go:429	mineOne: unexpectedly nil MiningBaseInfo for round 252970, off tipset 252969/{bafy2bzaceduumindh3htfg4owirchcchzo6xklvo42lqiphjabl7ad7leiogs,bafy2bzaceaus5qnfpm2rgpx6awxufx5zc66du4yvhnojh2zbwph47ozbrbmgi}

**Version (run `lotus version`):**
<pre>
Daemon:  1.9.0+git.ada7f97ba+api1.3.0
Local: lotus version 1.9.0+git.ada7f97ba
</pre>

Are the issue related to the warning in log?
Any suggestion on solve the issue?

**Comment by jennijuju** [comment]
the linked log should be unrelated. 
Search for logs  keyword in this method https://github.com/filecoin-project/lotus/blob/f9596dd730cdb70e3465450b66299606953c7785/extern/storage-sealing/states_sealing.go#L466 instead
